.mailmap

```diff
@@ -624,7 +624,8 @@ Varada Kari <varada.kari@sandisk.com>
 Varada Kari <varada.kari@sandisk.com> <kariraja@yahoo.com>
 Varada Kari <varada.kari@sandisk.com> <varada.kari@gmail.com>
 Varada Kari <varadaraja.kari@flipkart.com>
-Varsha Rao <varao@redhat.com>
+Varsha Rao <rvarsha016@gmail.com>
+Varsha Rao <rvarsha016@gmail.com> <varao@redhat.com>
```

Comment:
Should this not be one line?

Reply:
Done. In few places there multiple addresses like this
https://github.com/ceph/ceph/blob/52e28c4c390a9c098f3a4887e10701988b00fdee/.mailmap#L619-L626